app/views/candidate_interface/start_page/eligibility.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, title_with_error_prefix(t('page_titles.eligibility'), @eligibility_form.errors.any?) %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_start_path) %>
+<% content_for :before_content, govuk_back_link_to(govuk_apply_start_page) %>
```

Comment:
```suggestion
<% content_for :before_content, govuk_back_link_to(candidate_interface_create_account_or_sign_in_path) %>
```

I think this back link was actually wrong before, and should have taken users back to create account/sign in page, which is the previous page they would have seen. Somewhat academic as this page is now hidden behind a feature flag.

Reply:
@TobyRet Just this outstanding issue; back link on the eligibility page should take you back to create an account/sign in page.

Reply:
Ah sorry my bad. Fixed